ReactSkia/pluginfactory/RnsPluginFactory.cpp

```diff
@@ -41,12 +59,26 @@ std::unique_ptr<RNSApplicationManagerInterface> RnsPluginFactory::createAppManag
 }
 #endif // RNSP_APP_MANAGER
 
+#if ENABLE(RNSP_PLATFORM_MANAGER)
+std::unique_ptr<RNSPlatformManagerInterface> RnsPluginFactory::createPlatformManagerHandle(PlatformCallbackClient &client) const {
+  if(platformManagerCreator_ == nullptr)
+    return nullptr;
+  return std::unique_ptr<RNSPlatformManagerInterface>(platformManagerCreator_(client));
+}
+#endif // RNSP_PLATFORM_MANAGER
+
 RnsPluginFactory::~RnsPluginFactory() {
 #if ENABLE(RNSP_APP_MANAGER)
   if (appMgrPluginHandler_) {
     dlclose(appMgrPluginHandler_);
   }
 #endif // RNSP_APP_MANAGER
+
+#if ENABLE(RNSP_PLATFORM_MANAGER)
```

Comment:
One suggestion from my side : Can this Name RNSP -->to refer RNSPlugin be Changed to RNSPLUGIN itself, as this is used across different files, I feel readability would be more and would avoid confusion between "P" is for plugin or "platform"